package.json

```diff
@@ -36,9 +36,9 @@
     "atob": "^2.0.3",
     "babel-polyfill": "^6.23.0",
     "babel-runtime": "^6.11.6",
-    "bootstrap": "^3.3.6",
+    "bootstrap": "^4.1.3",
```

Comment:
This seems a bit intense to me... don't we have to change markup for this?

Reply:
Yeah, you're right, I'll revert that, I'm not going to dig into visual changes, at least for now ;)